src/graphql/data/Token.ts

```diff
@@ -19,6 +21,7 @@ const tokenQuery = graphql`
   query TokenQuery($contract: ContractInput!, $duration: HistoryDuration!) {
     tokens(contracts: [$contract]) {
       id @required(action: LOG)
+      decimals
```

Comment:
Would we want to add @required(action: LOG) (basically voids a returned token if decimals aren't present, so we don't have to null check for it)? I'm not sure how important decimal accuracy is or if the default is fine

Reply:
It's important to have decimal accuracy, but decimals are not required to be explicit for ERC-20s (see https://github.com/Uniswap/interface/pull/4988), so I don't think it's appropriate to require it. Instead, we should just use DEFAULT_ERC20_DECIMALS if it is not present.

Reply:
yes we dont need decimals for ERC 20, the default is 18 decimals, unless you use ERC20Detailed extension to specify a custom decimal number ( see here for documentation on diff between ERC20 and ERC20Detailed: https://docs.openzeppelin.com/contracts/2.x/api/token/erc20)